packages/react-components/src/components/SendBox.tsx

```diff
@@ -217,12 +242,36 @@ export const SendBox = (props: SendBoxProps): JSX.Element => {
   };
 
   /* @conditional-compile-remove(file-sharing) */
-  const renderFileUploadErrorMessage: JSX.Element = useMemo(() => {
+  const uploadInProgressErrorBar = useMemo(() => {
+    const renderError = !!(
+      props.activeFileUploads?.length &&
+      !props.activeFileUploads
+        .filter((fileUpload) => !fileUpload.errorMessage)
+        .every((fileUpload) => fileUpload.isUploaded)
+    );
+    return renderError && <SendBoxErrorBar message={strings.fileUploadsPendingError} />;
```

Comment:
Can we update latesterror on line 258 to fileUploadsPendingError rather than adding SendBoxErrorBar here.

Reply:
The logic is quite complex for these error states.
Upload pending error is only shown when someone tries to send a message as opposed to fileUpload error that should be visible as soon as it's in state. So `latestError` won't work here.